src/Connector/GceServiceAccountAuthentication.php

```diff
@@ -37,4 +38,18 @@ protected function authClient(): ClientInterface {
     return $sdk_connector->buildClient($this->getAuthHeader(), $this->getAuthServer());
   }
 
+  /**
+   * Returns the list of all Organizations available to the service account.
+   *
+   * @return \Apigee\Edge\Api\Management\Entity\OrganizationInterface[]
+   *   Returns the list of organizations the service account can access.
+   */
+  public function getAuthorizedOrganizations(): array {
+    /** @var \Drupal\apigee_edge\SDKConnectorInterface $sdk_connector */
+    $sdk_connector = \Drupal::service('apigee_edge.sdk_connector');
```

Comment:
Still, using dependency injection here would be useful, not just because the code could unit tested :)